components/properties/src/script.rs

```diff
@@ -7,45 +7,43 @@
 
 use crate::error::PropertiesError;
 use crate::props::Script;
+use crate::props::ScriptULE;
 use crate::provider::*;
 
-use crate::props::ScriptULE;
 use core::iter::FromIterator;
 use core::ops::RangeInclusive;
 use icu_collections::codepointinvlist::CodePointInversionList;
-use icu_collections::codepointtrie::CodePointTrie;
 use icu_provider::prelude::*;
-#[cfg(feature = "serde")]
-use serde::{Deserialize, Serialize};
-use zerovec::{ule::AsULE, VarZeroVec, ZeroSlice};
+use zerovec::{ule::AsULE, ZeroSlice};
 
 /// The number of bits at the low-end of a `ScriptWithExt` value used for
 /// storing the `Script` value (or `extensions` index).
 const SCRIPT_VAL_LENGTH: u16 = 10;
 
 /// The bit mask necessary to retrieve the `Script` value (or `extensions` index)
 /// from a `ScriptWithExt` value.
-const SCRIPT_X_SCRIPT_VAL: u16 = (1 << SCRIPT_VAL_LENGTH) - 1;
+pub(crate) const SCRIPT_X_SCRIPT_VAL: u16 = (1 << SCRIPT_VAL_LENGTH) - 1;
```

Comment:
Nit: You can undo some of these `pub(crate)` now that the code is back in this file

```suggestion
const SCRIPT_X_SCRIPT_VAL: u16 = (1 << SCRIPT_VAL_LENGTH) - 1;
```